src/toady/cli.py

```
@@ -612,28 +612,55 @@
+    "--yes",
+    is_flag=True,
+    help="Skip confirmation prompt for bulk operations",
+)
 @click.option(
     "--pretty",
     is_flag=True,
     help="Output in human-readable format instead of JSON",
 )
 @click.pass_context
-def resolve(ctx: click.Context, thread_id: str, undo: bool, pretty: bool) -> None:
+def resolve(
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)